### PR TITLE
Add demo access on the application page

### PR DESCRIPTION
**Pull request by martinsam**
unspecified
Add demo access on the app page and regenerate app.html :

### CODE REVIEW

File: src/apps.html [hunk 1]
```diff
@@ -32,16 +32,18 @@ <h1 class="introduction-title">{{__ 'apps title'}}</h1>
...
-      <div class="app-img">
-        <img src="../images/apps/calendar.svg"
+      <div>
+        <img src="../images/spinner.svg"
+             data-src="../images/apps/calendar.svg"
              style="background: #EC7E63"/>
       </div>
       <div>
-        <h3 class="app-title">Calendar</h3>
+        <h3 class="app-title">Calendar <a class="app-demo" title="Go to the calendar demo" href="https://demo.cozycloud.cc/#apps/calendar/">Demo</a></h3>
```
Comment (clochix) [inline src/apps.html]: Could you localize the link title, please? use {{__ 'string key'}} and add the key to src/locales/en.json

**Comment by clochix** [comment]
Thanks a lot @martinsam ☺
Pinging @frankrousseau for the review.
Refs #24

**Comment by frankrousseau** [comment]
I'll review it tomorrow morning.

File: utils/app.js [hunk 2]
```diff
@@ -57,6 +71,15 @@ client.get("repos/cozy/cozy-registry/contents/apps", (req, res, body) => {
       repoSlug = repoInfos[4].split('.')[0];
 
       var color = colorPicker.getColor(app.slug, 'cozy');
+
+      var demo_tag = '';
+      log.info(app.slug);
```
Comment (frankrousseau) [inline utils/app.js]: Can you remove this log?

File: utils/app.js [hunk 3]
```diff
@@ -57,6 +71,15 @@ client.get("repos/cozy/cozy-registry/contents/apps", (req, res, body) => {
...
+      if (repos_with_demo.hasOwnProperty(app.slug)){
+        var demo_url = repos_with_demo[app.slug];
+        var demo_title = `{{__ 'Go to the ${app.slug} demo' }}`;
+        var demo_tag = `<a class="app-demo" title="${demo_title}" href="${demo_url}">Demo</a>`;
```
Comment (frankrousseau) [inline utils/app.js]: Are you sure that this line is < to 80 chars?

File: utils/app.js [hunk 4]
```diff
@@ -8,6 +8,20 @@ var colorPicker = require('./color');
...
+  'calendar': 'https://demo.cozycloud.cc/#apps/calendar/',
+  'emails': 'https://demo.cozycloud.cc/#apps/emails/account/new',
+  'files': 'https://demo.cozycloud.cc/#apps/files/',
+  'blog': 'https://demo.cozycloud.cc/#apps/blog/',
+  'konnectors': 'https://demo.cozycloud.cc/#apps/konnectors/',
+  'pr-owm': 'https://demo.cozycloud.cc/#apps/pr-owm/',
+  'photos': 'https://demo.cozycloud.cc/#apps/photos',
+  'sync': 'https://demo.cozycloud.cc/#apps/sync/',
+  'term': 'https://demo.cozycloud.cc/#apps/term/',
+  'cozic': 'https://demo.cozycloud.cc/#apps/cozic/'};
```
Comment (frankrousseau) [inline utils/app.js]: Could you remove sync, term and cozic from this list?

File: utils/app.js (hunk 2 above)
Comment (martinsam) [inline utils/app.js]: Done

File: utils/app.js (hunk 3 above)
Comment (martinsam) [inline utils/app.js]: Done

File: utils/app.js (hunk 4 above)
Comment (martinsam) [inline utils/app.js]: Done

**Pull request merged**